### DISCUSSION TITLE
Can I count the stars of projects in organization?

### DISCUSSION THREAD

**Discussion by L2ncE** [anuraghazra/github-readme-stats#2183] [Q&A] [answered]
After I move the project to the organization, the star stats decrease.Is there any way to avoid the reduction?

**Comment by rickstaa** [comment, marked as answer]
@L2ncE Currently, we do not include stats of organization repositories in the cards. However, this is one of the most requested features (see #1). This feature has been implemented in #1122, but some things still must be fixed. However, you can already use this feature on your own fork by deploying your own Vercel instance (see https://github.com/anuraghazra/github-readme-stats#deploy-on-your-own-vercel-instance). You can follow the steps explained in https://github.com/anuraghazra/github-readme-stats/issues/1#issuecomment-981419174 to do this.
